options/options.go

```diff
@@ -57,6 +58,33 @@ func Load(log *ll.Entry, filepath string) (*DHCP, error) {
 	return parse(log, bytes)
 }
 
+// Parse a string as a CIDR or as an IP with implicit /24 prefix.
+func parseIP(log *ll.Entry, ipstr string) (*net.IPNet, error) {
+	// Parse as CIDR
+	ip, ipnet, err := net.ParseCIDR(ipstr)
+	if err == nil {
+		log.Infof("Parsed successfully as CIDR: %s", ipstr)
+		// Use IPNet to store the actual IP and netmask
+		ret := net.IPNet{
+			IP:   ip,
+			Mask: ipnet.Mask,
+		}
+		return &ret, nil
```

Comment:
There should be handrails preventing anything greater than a /30 from being returned as a valid IPNet.  These have minimal use case and would break the `gatewayFromIP` func

Reply:
I have added this restriction and tests, please check the latest commit.